.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+Thank you for contributing to this project! You must fill out the information below before we can review this pull request.
+-->
+
+### Why:
+
+This PR fixes: [issue link]
+
+### What's being changed:
+
+<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->
+
+### Type of change:
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Feature or enhancements (non-breaking change which adds functionality)
+- [ ] Documentation updates (non-breaking change which updates documents)
+
+### How Has This Been Tested?

```

Comment:
@Asmit2952 I think the testing justification is not mandatory for the documentation project. Maintainers/Reviewers can refer to the content itself to check it or view the Netlify preview link to verify front-end layout changes. What are your thoughts?

Reply:
Well this can be more of an additional self-reviewing check for contributors when they make a PR. This will make sure that everything is working on their end. Example PRs can be the one related to ORAS Navbar: #103 #106. We can add a comment under it saying `<!-- Remove this line if it's not relevant to your PR -->`.